README.md

```diff
@@ -60,6 +66,7 @@ The [WAI-ARIA](https://www.w3.org/TR/wai-aria-1.1/) spec is deliberately not dup
 
 ## Accessibility
 
+- Set `meta` tag in `html` to `content="width=device-width, initial-scale=1.0”`
```

Comment:
Don't most frameworks do this out of the box, and is this even required for modern browsers? Haven't set this myself for ages!

Reply:
Fair call. It's like a html 101 thing eh. Too obvious?

Reply:
Yeah I think so